vm-linux-helpers.cmake

```diff
@@ -52,6 +52,46 @@ function(AddFileToOverlayDir filename file_location root_location overlay_name)
     )
 endfunction(AddFileToOverlayDir)
 
+# Function to add a symbolic link to a relative path within the overlay
+# filename: The name of the link in the overlay
+# file_location: Absolute path to the file we are adding the overlay
```

Comment:
This comment still seems misleading - shouldn't it talk about a relative path within the overlay?

Reply:
Ah yup - I missed that one. Updated, and added an example use case in the comment.

Reply:
Thanks